src/openapi/paths/officialtexts/officialtexts.yml

```diff
@@ -1,7 +1,7 @@
-/officialdocuments:
+collection:
```

Comment:
Is that normal ? Should it be something like "/officialtexts" ?

Reply:
It can be anything, it's just a key after all. And super easier to reference. Should we do it in all the rest of openapi docs?